package.json

```diff
@@ -172,26 +174,23 @@
     ]
   },
   "apollo": {
-    "schemas": {
-      "default": {
-        "schema": "saleor/graphql/schema.graphql",
-        "endpoint": "http://localhost:8000/graphql/"
+    "client": {
+      "addTypename": true,
+      "includes": [
+        "saleor/static/dashboard-next/**/*.ts",
+        "saleor/static/dashboard-next/**/*.tsx"
```

Comment:
Can we also have `*.py` files here as well? We could enable autocompletion in tests where we have a lot of queries.

Reply:
That causes codegen to generate Typescript types for Python queries 😞